### ISSUE TITLE
Allow mark and copy of webforms during installation

### ISSUE THREAD

**Issue by wollmers** [RotherOSS/otobo#460]
Maybe browser specific (Safari).

It's possible to mark the labels in the webforms (left mouse key), but on key up it's unmarked.

Maybe a JavaScript event is too intrusive.

**Comment by svenoe** [comment]
Which forms are you speaking of? The customer interface, or the agent interface? For me it is working for both, although in the customer interface it is not so nice, yes, as the labels are responding to the mouse movement,...

**Comment by wollmers** [comment]
@svenoe "during installation" i.e. neither customer nor agent interface.
Don't remember exactly which ones or all.

**Comment by stefanhaerter** [comment]
I tried to investigate this. At the moment, I don't have a safari browser at hand, but I went with Firefox and Chromium on Linux. I had no problem marking the labels, plain text and headings (however, the headings looked a little odd when marked in Firefox). It was a bit difficult to mark button text and the text in the progress bar, but not impossible. The only thing I couldn't mark were the select options for the database type. Maybe someone with Safari could test this again?
